### PR TITLE
TW-1942: Change the design of log out dialog

### PR DESCRIPTION
**Pull request by hieutbui**
## Ticket
- #1942 

## Impact description
- Update style of logout confirm dialog follow new design.

## Test recommendations
- Open a dialog and test for: Adaptive, buttons function in all platforms.

## Resolved
- Web:

https://github.com/user-attachments/assets/eec15a99-3492-40a6-8bdb-8ef12230dce5

- Android:
- IOS:

### CODE REVIEW

**Comment by nqhhdev** [comment]
Screenshot 2024-07-16 at 14 27 49


Pls check pipeline

**Review by Te-Z — APPROVED** [review]

File: assets/l10n/intl_en.arb [hunk 1]
```diff
@@ -3082,5 +3082,6 @@
   "byContinuingYourAgreeingToOur": "By continuing, you're agreeing to our",
   "youDontHaveAnyContactsYet": "You dont have any contacts yet.",
   "loading": "Loading...",
-  "errorDialogTitle": "Oops, something went wrong"
+  "errorDialogTitle": "Oops, something went wrong",
+  "logoutDialogWarning": "You will lose access to encrypted messages. We recommend that you enable chat backups before loggingout"
```
Comment (KhaledNjim) [inline assets/l10n/intl_en.arb:3086]: there is a missing space between "logging" and "out."

File: lib/utils/dialog/twake_dialog.dart [hunk 2]
```diff
@@ -227,3 +231,180 @@ class ProgressDialog extends StatelessWidget {
...
+          type: MaterialType.transparency,
+          child: Container(
+            height: double.infinity,
+            width: double.infinity,
+            color: Colors.transparent,
+            child: Center(
+              child: Container(
+                constraints: BoxConstraints(
+                  maxWidth: responsiveUtils.isMobile(context)
+                      ? responsiveUtils.getSizeScreenWidth(context) - 48
```
Comment (sherlockvn) [inline lib/utils/dialog/twake_dialog.dart:271]: what if width of the screen is less than 48 ?

File: lib/utils/dialog/twake_dialog.dart [hunk 3]
```diff
@@ -227,3 +231,180 @@ class ProgressDialog extends StatelessWidget {
...
+                      offset: const Offset(0, 4),
+                      spreadRadius: 3,
+                    ),
+                    BoxShadow(
+                      color: Colors.black.withOpacity(0.3),
+                      blurRadius: 3,
+                      offset: const Offset(0, 1),
+                    ),
+                  ],
+                ),
```
Comment (sherlockvn) [inline lib/utils/dialog/twake_dialog.dart:290]: you should move it into style file

**Comment by hoangdat** [comment]
Figma discussion: https://www.figma.com/design/uLw3iWnOEGaRlJnfPz4Pv0?node-id=25506-130685&m=dev#965785114

**Comment by nqhhdev** [comment]
hi @hoangdat @KhaledNjim, pls help me review this PR

**Comment by hoangdat** [comment]
@KhaledNjim please review it

**Review by hoangdat — APPROVED** [review]

**Pull request merged**